convert2rhel/subscription.py

```diff
@@ -242,13 +246,33 @@ def install_rhel_subscription_manager():
         # When using the original system repos, we need YUM/DNF to expand the $releasever by itself
         set_releasever=False,
     )
-    if ret_code:
-        loggerinst.critical("Failed to install subscription-manager packages. See the above yum output for details.")
-    else:
```

Comment:
Just a nitpick. We could keep `if ret_code:` and simply remove the else statement since critical is a SysExit